x/gov/module.go

```diff
@@ -176,8 +190,17 @@ func provideModule(in govInputs) govOutputs {
 		kConfig.MaxMetadataLen = in.Config.MaxMetadataLen
 	}
 
-	k := keeper.NewKeeper(in.Cdc, in.Key, in.Subspace, in.AccountKeeper, in.BankKeeper, in.StakingKeeper, in.MsgServiceRouter, kConfig)
-	m := NewAppModule(in.Cdc, k, in.AccountKeeper, in.BankKeeper)
+	k := keeper.NewKeeper(
+		in.Cdc,
+		in.Key,
+		in.AccountKeeper,
+		in.BankKeeper,
+		in.StakingKeeper,
+		in.MsgServiceRouter,
+		kConfig,
+		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
```

Comment:
We've changed it to have it non hard-coded anymore: c.f. https://github.com/cosmos/cosmos-sdk/pull/12660